R/lineage.R

```diff
@@ -103,11 +69,10 @@ LineageTree <- function(object, clonotype, regions = "V", reference, chain = c("
 #' Build a lineage tree for the given sequences
 #'
 #' @param sequences Named vector of sequences
-#' @param distance.method Method to use to compute distances between the sequences. Default = lv (Levenshtein)
 #' @param root Should the tree be rooted. Default = FALSE
 #' @param outgroup Outgroup to root the tree
 
-BuildLineageTree <- function(sequences, distance.method = "lv", outgroup = NULL, root = F) {
+BuildLineageTree <- function(sequences, outgroup = NULL, root = F) {
```

Comment:
I would prefer to keep the distance method on this function. It is now impossible to change the distance metric when making the tree